### ISSUE TITLE
Migration Requirement 1 (suggestion): ToDo List

### ISSUE THREAD

**Issue by cevap** [cevap/ion#19]
We need a list of tasks required to prepare all required data for the migration process.

I suggest to start simple:

1. Definition of CE goals

1. Full documentation about involved code/changes parts and all it's childs as well as parents

1. Based on documentation, definition of tasks/requirements/dependencies to be done/prepared

1. Define workflow and Tools required for previous step/s, as well as their documentation

1. Migration process

**Comment by mitchellcash** [comment]
Sorry I might have missed the conversation on Slack, what migration?

**Comment by cevap** [comment]
yes, you missed. We speak about migration to latest 0.14.99+ core. For that quite a lot has to be recoded.
